tests/Sdk/Acceptance/Presentation/RestApi/Controller/v1/RunTaskControllerCest.php

```diff
@@ -34,15 +36,51 @@ class RunTaskControllerCest
      *
      * @return void
      */
-    public function iSeeJsonResponseAfterCallHelloWorldEndpoint(AcceptanceTester $I): void
+    public function iSeeJsonResponseAfterCallRunTaskEndpoint(AcceptanceTester $I): void
     {
-        $I->sendPost(static::ENDPOINT, ['world' => 'World', 'somebody' => 'World']);
+        $I->haveHttpHeader('Content-Type', 'application/json');
+        $I->sendPost(static::ENDPOINT, [
+            OpenApiField::DATA => [
+                OpenApiField::TYPE => RunTaskController::TYPE,
+                OpenApiField::ID => 'hello:world',
+                OpenApiField::ATTRIBUTES => [
+                    'world' => 'World', 'somebody' => 'World',
+                ],
+            ],
+        ]);
         $I->seeResponseCodeIs(Response::HTTP_OK);
 
         $I->seeResponseContainsJson([
-        'messages' => [
-            'Executing stage: hello',
-            'hello \'World\' \'World\'',
-        ]]);
+            OpenApiField::DATA => [
+                OpenApiField::ID => 'hello:world',
+                OpenApiField::TYPE => RunTaskController::TYPE,
+                OpenApiField::ATTRIBUTES => [
+                    'messages' => [
+                        'Executing stage: hello',
+                        'hello \'World\' \'World\'',
+                    ],
+                ],
+            ],
+        ]);
+    }
+
+    /**
+     * @param \SprykerSdk\Sdk\Tests\AcceptanceTester $I
+     *
+     * @return void
+     */
+    public function iSeeBadRequestAfterCallRunTaskEndpoint(AcceptanceTester $I): void
+    {
+        $I->haveHttpHeader('Content-Type', 'application/json');
+        $I->sendPost(static::ENDPOINT, [
+            OpenApiField::DATA => [
+                OpenApiField::TYPE => RunTaskController::TYPE,
+                OpenApiField::ID => 'hello:world',
+                OpenApiField::ATTRIBUTES => [
+                    'world' => 'World',
+                ],
+            ],
+        ]);
+        $I->seeResponseCodeIs(Response::HTTP_BAD_REQUEST);
```

Comment:
Would be nice to check response body as well

Reply:
Fixed